lib/matplotlib/patches.py

```diff
@@ -1449,6 +1449,39 @@ def get_center(self):
 
     center = property(get_center, set_center)
 
+    def set_width(self, width):
+        '''
+        Set the width of the ellipse
+        
+        Parameters
+        ----------
+        width : float
+        '''
+        self.width = width
+        self.stale = True
```

Comment:
This suggests that width should be transformed into a property (like center) so that assignment to it also marks the ellipse as stale.  Making it a property would also at least make sure that the code paths are exercised.

Alternatively width could even be turned into a property (to ensure staleness marking) without even having public getters and setters, but the getters and setters do make the API more uniform (given that they already exist for center).

Reply:
True, I was not familiar with properties but I can make a second commit in the PR for that as soon as I recover a working machine 😅